app/controllers/api/v1/salads_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::SaladsController < ApplicationController
+
+  def index
+    salads = Salad.all
+    render json: salads
+  end
+end
```

Comment:
Hmm... sorry, but what is this about? Why would you create a separate class for a salad? And in next PR a drink?  Are these not variants of a dish?

Reply:
No, there will be few different types of salads and drinks. And with the way they are going to be displayed its easier to keep them apart from the dishes.